masterlist.yaml

```
@@ -8239,6 +8239,28 @@ plugins:
       - crc: 0x6F9705F8
         util: 'TES4Edit v4.0.3h (Hotfix 1)'
 
+  - name: 'Ayleid Loot EXtension( - OOO Dungeons)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/22795/'
+        name: 'Ayleid Loot EXtended'
+  - name: 'Ayleid Loot EXtension.esp'
+    msg:
+      - <<: *patchProvided
```

The patch is in the mod's archive, so `*patchIncluded` is more appropriate.

```suggestion
      - <<: *patchIncluded
```